### PR TITLE
Add 1.x EOL and 2.x migration docs

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Comment by jlipps** [comment]
@mykola-mokhnach i pulled in your commit and worked on it some more. we can probably close that other PR?

File: docs/en/about-appium/1.x-eol.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Appium 2.0 is currently in beta and is ready for your use and testing. Please begin the process of
+ensuring your test suites, or any other software built on Appium, is compatible with Appium 2.0.
+
+The tentative official release date of Appium 2.0 is *September 15, 2021*. After the release of
+Appium 2.0 (whether it is on that date or not), we will not be making any feature additions to the
+Appium 1.x server. We will however make the following kinds of updates for 6 months following the
+release of Appium 2.0:
+
+* Feature updates and bug fixes to core drivers (XCUITest, UiAutomator2, Espresso). For the first
```
Comment (mykola-mokhnach) [inline docs/en/about-appium/1.x-eol.md:26]: is this bullet-points formatting expected?

File: docs/en/about-appium/1.x-eol.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+changes, their rationale, and what you will need to do to keep your tests running well.
+
+## Transition Plan
+
+Appium 2.0 is currently in beta and is ready for your use and testing. Please begin the process of
+ensuring your test suites, or any other software built on Appium, is compatible with Appium 2.0.
+
+The tentative official release date of Appium 2.0 is *September 15, 2021*. After the release of
+Appium 2.0 (whether it is on that date or not), we will not be making any feature additions to the
+Appium 1.x server. We will however make the following kinds of updates for 6 months following the
```
Comment (KazuCocoa) [inline docs/en/about-appium/1.x-eol.md:23]: Btw, the release date of Appium 1.22 depends on iOS15/Xcode 13 for now. So, we can address that the 1.22 will be the last release as 1.x here as well. (The next 1.x will be only critical issue fixes then)

**Review by KazuCocoa — COMMENTED** [review]
👍 
After this merge, I'd like to add a link to this 1.x EOL in https://github.com/appium/appium/pull/15640

**Comment by jlipps** [comment]
OK, I think this is ready to merge for now.

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**